contextPlus.js

```diff
@@ -4,78 +4,80 @@ const contextMenuContainers = {
       return;
     }
 
-    const contextualIdentities = await browser.contextualIdentities.query({});
-    const parentId = browser.contextMenus.create({
-      id: "moveContext",
-      title: "Move to Context",
-      contexts: ["tab", "page"]
-    });
+    const defaultCookieStoreId = "firefox-default";
 
+    const contextualIdentities = await browser.contextualIdentities.query({});
     const contextStore = contextualIdentities.reduce((store, context) => {
-      return Object.assign({}, store, {
-        [`contextPlus-${context.name}`]: context.cookieStoreId
-      });
+      return Object.assign(
+        { "contextPlus-default": defaultCookieStoreId },
+        store,
+        { [`contextPlus-${context.name}`]: context.cookieStoreId }
+      );
     }, {});
```

Comment:
Apparently Firefox supports spreads in object literals now. Not sure how I missed that!

```javascript
contextualIdentities.reduce((store, context) => ({
  ...store,
  [`contextPlus-${context.name}`]: context.cookieStoreId
}), {'contextPlus-default': defaultCookieStoreId});
```